src/main/java/cs2103/v15_1j/jimjim/AddTaskCommand.java

```diff
@@ -19,9 +19,9 @@ public String undo(List<TaskEvent> displayList, Storage storage, Searcher search
 
 	@Override
 	public String execute(List<TaskEvent> displayList, Storage storage, Searcher searcher) {
-	    if (storage.create(task)) {
+		displayList.add(task);
```

Comment:
One contains item for display e.g. search result one is for the internal working of the logic